### ISSUE TITLE
Free Seconds Counters

### ISSUE THREAD

**Issue by leandronvoip** [iNextrix/ASTPP#459]
Hi,

I found the follow errors:

The customers has a free call package.

If the call for landline or mobile line, the counters will increase normally.

If the call is for a DID configured internal or to a another account, the counters will make double. Example: Call is 10 seconds to a DID, the counter table will register 20 seconds.

**Comment by devangn** [comment]
Hello,

Well, that's because package is assigned to rate group and the DID call affecting to two accounts which are having same rate group with package. That's the reason its count 10 secs twice, once for both account, because you may have select 'Application For' 'Both' in package config.

**Comment by leandronvoip** [comment]
Yes. Really this test I use both on packages. Are working. Can close the issue.

**Comment by leandronvoip** [comment]
I put only outbound and check another error: the call don't complete how free call. Only complete if has balance.

Example:

Package has 60 seconds of free calls to prefix 55.
I put package with both (he work and debit double when calling a internal did number).
I put package with inbound (he work for internal did number)
I put package with outbound (he don't work for internal did number without balance). With balance he make a free call.
